lib/index.js

```diff
@@ -459,6 +460,49 @@ const commands = {
 
         callback(null, textify.reverse(args));
     },
+    roll: function(args, callback) {
+	const dice = new roll();
+	const orig = args;
+	const show = orig.toString().endsWith('-show');
+	args = args.replace('-show','').trim();
+
+        if (!args) {
+            args = 'd100';
```

Comment:
Depends on the user and context.  When I think of "rolling the dice", I think of 2d6 over d6.

The reason I went with d% (d100) is because I think it has a much higher probability of being used day to day than a d6 or 2d6 would, and people who already know notation will probably specify it anyways if they wanted a d6.

I can change it, but if I do, I'd like to make a /random command also that defaults 1-100, as I see that getting more use.